app/controllers/verify/review_controller.rb

```diff
@@ -23,7 +23,13 @@ def new
       @idv_params = idv_params
       analytics.track_event(Analytics::IDV_REVIEW_VISIT)
 
-      flash.now[:success] = flash_message_content
+      usps_mail_service = Idv::UspsMail.new(current_user)
+      flash_now = flash.now
+      if usps_mail_service.mail_spammed?
+        flash_now[:error] = t('idv.errors.mail_limit_reached')
```

Comment:
Codeclimate reports this branch is untested, can we add a spec to the controller that covers this?